README.md

```diff
@@ -9,84 +12,79 @@ Documentation
 High level documentation and usage examples are located
 [here](http://docs.mongodb.org/ecosystem/drivers/ruby/).
 
-API docs can be found [here](http://api.mongodb.org/ruby/).
+API documentation can be found [here](https://api.mongodb.org/ruby/).
 
 
-Support & Feedback
-------------------
+Support
+-------
+
+Commercial support for the driver is available through the
+[MongoDB Support Portal](https://support.mongodb.com/).
 
-For issues, questions or feedback related to the Ruby driver, please look into
-our [support channels](https://docs.mongodb.com/manual/support). Please
-do not email any of the Ruby developers directly with issues or
-questions - you're more likely to get an answer quickly on the
+For questions, discussions or general technical support, please visit the
 [MongoDB Community Forum](https://developer.mongodb.com/community/forums/tags/c/drivers-odms-connectors/7/ruby-driver).
 
+Please see [Technical Support](https://docs.mongodb.com/manual/support/) page
+in the documentation for other support resources.
+
 
 Bugs & Feature Requests
 -----------------------
 
-Do you have a bug to report or a feature request to make?
+To report a bug in the driver or request a feature specific to the Ruby driver:
 
-1. Visit [our issue tracker](https://jira.mongodb.org) and login (or create an account if necessary).
-2. Navigate to the [RUBY](https://jira.mongodb.org/browse/RUBY) project.
-3. Click 'Create Issue' and fill out all the applicable form fields.
+1. Visit [our issue tracker](https://jira.mongodb.org/) and login
+(or create an account if you do not have one already).
+2. Navigate to the [RUBY project](https://jira.mongodb.org/browse/RUBY).
+3. Click 'Create Issue' and fill out all of the applicable form fields.
 
-When reporting an issue, please keep in mind that all information in JIRA for all driver projects (ex. RUBY, CSHARP, JAVA) and the Core Server (ex. SERVER) project is **PUBLICLY** visible.
+When creating an issue, please keep in mind that all information in JIRA
+for the RUBY project, as well as the core server (the SERVER project),
+is publicly visible.
 
-**PLEASE DO**
+**PLEASE DO:**
 
-* Provide as much information as possible about the issue.
-* Provide detailed steps for reproducing the issue.
-* Provide any applicable code snippets, stack traces and log data.
-* Specify version information for the driver and MongoDB.
+- Provide as much information as possible about the issue.
+- Provide detailed steps for reproducing the issue.
+- Provide any applicable code snippets, stack traces and log data.
```

Comment:
How would you feel about specifically asking for "copy-pastable" code snippets (i.e. not screenshots)?

Reply:
I am not against it but also I haven't seen anyone posting screenshots in our interaction media.

Reply:
It happened once but I agree that it's not a common occurrence. 👍